src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -23,6 +23,12 @@
 	public Integer mysqlPort;
 	public String  mysqlUser;
 	public String  mysqlPassword;
+
+	public String mysqlSchemaHost;
+	public Integer mysqlSchemaPort;
```

Comment:
sure, go for it.  the other thing I was thinking about was that maybe we could accept uris: 

```
  --mysql='mysql://user:password@server:port'
  --schema_mysql='...'
```